mobify/astro-scaffold
# HYB-690: Adding double icons plugin and search icon

Adding Double Icons Plugin and Search icon on the right of the header bar

JIRA: https://mobify.atlassian.net/browse/HYB-690
Linked PRs: https://github.com/mobify/astro/pull/444

## Changes
- Add double icon plugin as custom plugin in scaffold (created in thinkgeek and matalan)
- Create a double icon controller which is initialized with functions which know how to generate the two plugins (needed when navigating)
- Integrate the `DoubleIconController` into the `NavigationHeaderController`
- Add rpc method to update the counter badge count


## How to test-drive this PR
- Link to `astro` branch `hyb-690-expose-auto-layout-extensions`
- Run app in ios and android
- Ensure that both the search icon and the cart icon appear  in the header (on right hand side)
- Ensure when you navigate the icons remain in the header

## TODOS:
